operations/pyroscope/relabeling_test.go

```diff
@@ -1,230 +0,0 @@
-package pyroscope
-
-import (
-	"context"
-	"fmt"
-	"os"
-	"testing"
-	"time"
-
-	"github.com/go-kit/log"
-	"github.com/prometheus/common/model"
-	"github.com/prometheus/prometheus/discovery/targetgroup"
-	"github.com/prometheus/prometheus/model/labels"
-	"github.com/stretchr/testify/require"
-	"gopkg.in/yaml.v3"
-
-	"github.com/grafana/pyroscope/pkg/agent"
-)
-
-func Test_Relabeling(t *testing.T) {
```

Comment:
Wonder if we can test the river config in a similar way

Reply:
Not sure we could lint the river file may be.

Reply:
I added a test that verify the river file is valid and formatted.